tests/src/test/scala/com/nvidia/spark/rapids/JsonScanRetrySuite.scala

```diff
@@ -29,7 +29,7 @@ class JsonScanRetrySuite extends RmmSparkRetrySuiteBase {
 
     val cudfSchema = GpuColumnVector.from(StructType(Seq(StructField("a", IntegerType),
       StructField("b", IntegerType))))
-    val opts = JSONOptions.builder().withLines(true).build()
+    val opts = GpuJsonReadCommon.baseCudfJsonOptionsBuilder().withLines(true).build()
```

Comment:
`line(true` is default and the plugin relies on it. So probably we should better set `withLines(true)` in `baseCudfJsonOptionsBuilder` instead.

Reply:
I think we should do that in a separate PR. I'd like to keep the change of this PR minimized.

Reply:
But you already changed the `cudfJsonOptions` function right? So adding one more change like that would not be different IMO.

Reply:
I am modifying the code in this function without functionality change. Your suggestion changes its behavior which may or may not have an impact on other tests. Given that this test failure is blocking all PRs, I think it's better to make that change in a different PR without unnecessarily blocking others further.

Reply:
Created https://github.com/NVIDIA/spark-rapids/issues/11899.